h2o-py/h2o/frame.py

```diff
@@ -2184,8 +2184,32 @@ def get_frame_data(self):
         >>> iris = h2o.import_file("http://h2o-public-test-data.s3.amazonaws.com/smalldata/iris/iris_wheader.csv")
         >>> iris.get_frame_data()
         """
-        return h2o.api("GET /3/DownloadDataset", data={"frame_id": self.frame_id, "hex_string": False, "escape_quotes" : True})
+        return h2o.api(
+            "GET /3/DownloadDataset", 
+            data={"frame_id": self.frame_id, "hex_string": False, "escape_quotes": True}
+        )
 
+    def save(self, path, force=True):
```

Comment:
regarding the load/save naming - is it appropriate for the native format? also for binary models, we have importModel/exportModel

Reply:
actually we have `h2o.load_model` and `h2o.save_model` in py, I am surprised we dont have `model.save()` that would the the only discrepancy with this API, should I move `frame.save()` to `h2o.save_frame()`? having an instance method on frame seemed more natural